CHANGELOG.md

```diff
@@ -93,6 +95,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
      `utils.coords.update_longitude`
    * Fixed pysat_testmodel Instrument longitude range
    * Corrected link to testing badge in README.md
+   * `from_os` now always returns a sorted Series.
+   * Improved robustness of parsing delimited files.
```

Comment:
Is this improvement really a bug fix or an enhancement?

Reply:
Moved the robustness comment.